guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,49 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== New All Hosts index page improvements ===
+
+The experimental new All Hosts index page gets some major improvements with this release. To use the new page by default, in Settings, set Show new host overview page to Yes.
```

Comment:
We put sentences on their own lines. Quoting https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#pull-request-checklist
> I put each sentence to its own line.

See https://sembr.org/ for more.